From 10d43ce6aea0a041a776445783b4e1ca09651f57 Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Tue, 4 Dec 2018 16:20:24 +0100
Subject: [PATCH] JDBC: Drop unneeded JAVAH variable check

Commit b8d0997 dropped javah detection from java.m4, but left in a JAVAH
variable check which broke the build, as nothing ever sets the variable
any more.
---
 m4/java.m4 | 1 -
 1 file changed, 1 deletion(-)

diff --git a/m4/java.m4 b/m4/java.m4
index f4e64e9af..e659a1bec 100644
--- a/m4/java.m4
+++ b/m4/java.m4
@@ -116,7 +116,6 @@ m4_define([_JAVA_CHECK_INTERNAL],
 	have_all_java=yes
 	if test -z "$JAVA_PROG"; then have_all_java=no; fi
 	if test -z "$JAVAC"; then have_all_java=no; fi
-	if test -z "$JAVAH"; then have_all_java=no; fi
 	if test -z "$JAR"; then have_all_java=no; fi
 	if test ${have_all_java} = no; then
 	    AC_MSG_WARN([one or more Java tools are missing (JRE is not sufficient)])

